### DISCUSSION TITLE
How to display/use delete button on preview?

### DISCUSSION THREAD

**Discussion by sheckathorne** [inventare/django-image-uploader-widget#86] [Q&A] [answered]
First, thanks for this great app, it looks so much better than the Django uploader.

How can you enable the delete button in the preview? I've read through the docs and don't find it mentioned. I looked through the source code a bit and tried this on my form which didn't seem to work:
```
        widgets = {
            'logo': ImageUploaderWidget(attrs={"data-candelete": "true"}),
        }
```

Thanks!
<img width="539" alt="Screenshot 2023-01-13 at 4 58 46 AM" src="https://user-images.githubusercontent.com/21269876/212304459-bfc35c20-64d1-4218-b85e-e6c480961023.png">

**Comment by EduardoJM** [comment, marked as answer]
Thanks for the comment. The delete button is shown when the field is not required into the django form. To made the field not required, add `null=True, blank=True` to the ImageField, like it:

```python
class TestNonRequiredInlineItem(models.Model):
    # ...
    image = models.ImageField('Image', null=True, blank=True)

```
